client/state/preferences/selectors.js

```diff
@@ -46,5 +47,5 @@ export const preferencesLastFetchedTimestamp = ( state ) => state.preferences.la
  * @returns {boolean}       Whether preferences have been received
  */
 export function hasReceivedRemotePreferences( state ) {
-	return !! state.preferences.remoteValues;
+	return !! state.preferences.remoteValues || config.isEnabled( 'is_running_in_jetpack_site' );
```

Comment:
All JITM are hidden if the value is false, so we test whether it's odyssey here.